tests/checks/integration/test_docker_daemon.py

```diff
@@ -73,9 +73,9 @@ def mock_get_info_no_data(self):
     def mock_get_info_invalid_values(self):
         return {
             'DriverStatus': [
-                ['Metadata Space Available', '9 MB'],
-                ['Metadata Space Total', '10 MB'],
-                ['Metadata Space Used', '11 MB'],
+                ['Metadata Space Available', '0 MB'],
```

Comment:
why do we set values to 0 ? Doesn't that broke `test_devicemapper_invalid_values` test ?